esi_leap/tests/db/sqlalchemy/test_api.py

```diff
@@ -259,4 +266,122 @@ def test_offer_get_by_name(self):
 
     def test_offer_get_by_name_not_found(self):
 
-        assert api.offer_get_by_uuid('some_uuid') is None
+        self.assertEqual(api.offer_get_by_name('some_name'), [])
+
+    def test_offer_destroy(self):
+
+        offer = api.offer_create(test_offer_2)
+        api.offer_destroy(offer.uuid)
+        self.assertEqual(api.offer_get_by_uuid('offer_2'), None)
+
+    def test_offer_destroy_not_found(self):
+
+        self.assertEqual(api.offer_get_by_uuid('offer_4'), None)
+
+    def test_offer_update(self):
+
+        offer = api.offer_create(test_offer_3)
+        o = api.offer_create(test_offer_2)
```

Comment:
You don't need to actually create this offer; you can just use test_offer_2['start_time'] below.

Reply:
ok ill change that around.

Reply:
i needed to create the offer in order to create the res to check the update against

Reply:
You need to create test_offer_3, but you don't need to create test_offer_2, right?

Reply:
oh i see now.